src/components/core/CardWrapper/CardWrapper.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC } from 'react';
```

Comment:
The name `CardWrapper` doesn't sound correct. What about `CardContainer` or `EntityCardsContainer` if it will be used only for `EntityCard`-s?